src/TLuaInterpreter.cpp

```diff
@@ -208,11 +208,11 @@ TLuaInterpreter::~TLuaInterpreter()
 //
 // The "isOptional" parameter is optional, and will default to not-optional parameters! :)
 //
-// See also: getVerifiedString, getVerifiedInt, getVerifiedFloat, announceWrongArgumentType
+// See also: getVerifiedString, getVerifiedInt, getVerifiedFloat, errorArgumentType
 bool TLuaInterpreter::getVerifiedBool(lua_State* L, const char* functionName, const int pos, const char* publicName, const bool isOptional)
 {
     if (!lua_isboolean(L, pos)) {
```

Comment:
This breaks `tempTimer(1, function() display("hello!") end)` where the last argument is optional

Reply:
and the error message reads:
`<tempTimer: bad argument #3 type (repeating as boolean is optional, got no value!)>`

Reply:
fixed